packages/EasyErrorHandler/src/Bridge/Symfony/Resources/config/api_platform_builders.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use EonX\EasyErrorHandler\Bridge\BridgeConstantsInterface;
+use EonX\EasyErrorHandler\Bridge\Symfony\Builder\ApiPlatformBuilderProvider;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services
+        ->set(ApiPlatformBuilderProvider::class)
+        ->arg('$keys', '%' . BridgeConstantsInterface::PARAM_RESPONSE_KEYS . '%');
+
+    // Drop tags of ValidationExceptionListener
+    $services
+        ->set('api_platform.listener.exception.validation')
+        ->synthetic(true);
```

Comment:
I'm not sure about that, I had issues in the past because a `synthetic` service must be initialised so if something else needs it, it might break the application...

Reply:
I didn't find any other ways to remove tags from this service. There was an another option  - define it as abstract (`->abstract(true)`) but I didn't tested this solution.

Reply:
I've made it abstract and what I've gotten:
![image](https://user-images.githubusercontent.com/1703419/104906474-cd571400-5994-11eb-9927-860a34bb6ea3.png)

As you can see tags and a class was removed.

Reply:
Btw, it's really close to how it was with synthetic:
![image](https://user-images.githubusercontent.com/1703419/104906861-4bb3b600-5995-11eb-868e-0b09fa7f6426.png)

Reply:
@itorgov did you try this solution in `payment-gateway-php` project to see if it works?

Another cleaner solution would be to create a compiler pass that completely remove this definition from the container, more verbose but I am sure it works.

Reply:
Finally I managed to remove that listener by using a compiler pass. Thanks for the suggestion!